pandas/tests/groupby/transform/test_transform.py

```diff
@@ -697,6 +697,53 @@ def test_groupby_cum_skipna(op, skipna, input, exp):
     tm.assert_series_equal(expected, result)
 
 
+@pytest.fixture
+def floating():
+    return Series(np.random.randn(10))
+
+
+@pytest.fixture
+def floating_missing(floating):
+    floating_miss = floating.copy()
+    floating_miss.iloc[2:7] = np.nan
+    return floating_miss
+
+
+@pytest.fixture
+def strings():
+    return list("abcde") * 2
+
+
+@pytest.fixture
+def strings_missing(strings):
```

Comment:
if floating, floating_missing, strings, string_missing are only used in `frame`, then id rather they be defined inside frame than as separate fixtures.  not a huge deal

Reply:
Good point. Yeah they are only used there so I can move them to `frame`